src/client/shared/ui/ui-sidebar/styles/sidebar-styles.scss

```
@@ -0,0 +1,116 @@
+  .group[data-side='right'] & {
+    transform: rotate(180deg);
+  }
+
+  .group[data-variant='floating'] &,
+  .group[data-variant='inset'] & {
+    .group[data-collapsible='icon'] & {
+      width: calc(var(--sidebar-width-icon) + 1rem);
+    }
+  }
+
+  .group:not([data-variant='floating']):not([data-variant='inset']) & {
```

Stylelint (17.6.0)

[error] Expected complex :not() pseudo-class notation (selector-not-notation)

(selector-not-notation)